### ISSUE TITLE
Need a way to disable / move / hide exile window

### ISSUE THREAD

**Issue by Greeneye0** [magefree/mage#1115]
These keep popping up right over the opponents health.  This is frustrating to play with.

Can we have exiles only show up if I click the exile icon, much like the graveyard?

I know that you would have to show the source of the exile somehow, but I'm sure you can think of something :)

![image](https://cloud.githubusercontent.com/assets/11687236/8660518/a49f5c2c-296e-11e5-9df4-303cbff8e165.png)

**Comment by LevelX2** [comment]
Move it by dragging it in the title bar. 
You can resize it.
You can minimize it too.

Or do you only mean the position it appears the first time in a game?

**Comment by Greeneye0** [comment]
Yes, but it keeps comming up over and over for every exile.  Sometimes and exile will cause all of the exile windows to need to be minimized again.

Even moving the position in game would be better than what it is now.

Try playing an exile heavy deck and you will see what I mean.
